src/components/Myprofile.js

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+import { ListGroup } from 'react-bootstrap';
+/* import './Myprofile.css'; */
+
+function Myprofile() {
+  const missions = useSelector((state) => state.mission);
+  const Missions = missions.filter((mission) => mission.reserved);
```

Comment:
- [ ] [Optional] Please consider changing the way you named the `Missions` variable because by convention, your variable names shouldn't start with uppercase. You can call it `missionsList`.
- [ ] [Optional] Same goes for the `Rockets` variable. You can call it `rocketsList` instead.
**NB** Having said that, I noticed that the reason you did that is because of your 2-step process of getting the missions from the store and filtering. This can be done in one step like this: 
```
const missions = useSelector((state) => state.missions.filter((mission) => mission.reserved))
```

Reply:
@codepantha  thanks for review the variable uppercase my bad I change it 
and I used the line of code you sent .